tests/codemods/test_remove_debug_breakpoint.py

```diff
@@ -0,0 +1,88 @@
+from core_codemods.remove_debug_breakpoint import RemoveDebugBreakpoint
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestRemoveDebugBreakpoint(BaseCodemodTest):
+    codemod = RemoveDebugBreakpoint
+
+    def test_name(self):
+        assert self.codemod.name() == "remove-debug-breakpoint"
+
+    def test_builtin_breakpoint(self, tmpdir):
+        input_code = """\
+        def something():
+            var = 1
+            breakpoint()
+        something()
+        """
+        expected = """\
+        def something():
+            var = 1
+        something()
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_builtin_breakpoint_multiple_statements(self, tmpdir):
+        input_code = """\
+        def something():
+            var = 1
+            print(var); breakpoint()
+        something()
+        """
+        expected = """\
+        def something():
+            var = 1
+            print(var); 
```

Comment:
this line is why I added this file to exclude removing trailing whitespace in pre-commit. There is a trailing space after `;`

Reply:
We might just want to omit test code from this check entirely.

Reply:
Sorry I mean we could omit the entire test directory from this pre-commit rule.